platform/android/java/lib/src/org/godotengine/godot/GodotLib.java

```diff
@@ -295,6 +295,22 @@ public static void calldeferred(long p_id, String p_method, Object[] p_params) {
 	 */
 	public static native void onRendererPaused();
 
+	/**
+	 * Invoked when the app is paused to release camera resources.
+	 */
+	public static native void onCameraPause();
+
+	/**
+	 * Invoked when the app is resumed to restore camera resources.
+	 */
+	public static native void onCameraResume();
```

Comment:
Instead of adding new methods, use `onRendererResumed()` and `onRendererPaused()` instead to notify the camera logic that it should resume / pause.